tox.ini

```diff
@@ -1,21 +1,26 @@
 [tox]
 envlist =
-    py{26}-django{15,16},
-    py{27,33,34}-django{15,16,17,18},
+    py{26,27,32,33}-django{15,16},
+    py{27,32,33,34}-django{17,18},
 
 [testenv]
+changedir = demo_project
 commands =
-    coverage run setup.py test
-    coveralls
-basepython =
-    py26: python2.6
-    py27: python2.7
-    py33: python3.3
-    py34: python3.4
+    python manage.py syncdb --noinput
+    python manage.py validate
+    # TODO: for Django>=1.7 execute instead: ===
```

Comment:
We should probably drop support for Django < 1.7. Quite a few projects are doing this, and I think it makes sense.

`makemigrations / migrate` replaces `syncdb / migrate` (and South) starting with Django 1.7, so there needs to be no distinction in the setup of (demo) projects.